### DISCUSSION TITLE
will there be a future Linux installer of all distributions without a compiler

### DISCUSSION THREAD

**Discussion by Darkinggq** [o3de/o3de#1935] [General]
Example of Godot Engine and Unity they install a program for Linux no compiler to quickly launch the editor on Linux it will be easier

looking forward to #746 

despite the market share of Linux 6% + to the whole world, I really love Linux ))

**Comment by slapin** [comment]
Would be happier to get more or less clear Linux build first, then installer will be a piece of cake. Also the market share depends on availability of professional software - the more the better.

**Comment by amznestebanpapp** [comment]
There are plans, I think we still don't know when we will start and how long it will take. But the hope is to add an installer for macOS and for Linux. We also will eventually add installers for target platforms, like to be able to build for Android in Windows (so that installer will have the o3de SDK for Windows and for Android, ideally, the same installer and you pick what you want to install).

As @slapin states, first we need to get the Editor/Launcher/AP up. After that, we need to check that the "install" target produces a proper image. Then we need to investigate an implement the proper installer generator (we use CPack). 

@amzn-pratikpa can those tasks be migrated to github?

**Reply by lemonade-dm** [reply]
One thing to remember, is that a lot of Linux installation goes through package manager, which is a more familiar workflow to users.
Some tools like VirtualBox does have a .sh script that can be used for installation, but it also has a package that is available in the OS package Manager. 
Perhaps an install layout using the engine's CMake INSTALL target and then combining that with a shell script that runs as a post install step of the package installation.
The Package Manager would also allow adding dependencies on other required packages such as libffi6 and cmake.
